README.md

```diff
@@ -5,13 +5,21 @@ Reads Deye solar inverter metrics using Modbus over TCP and publishes them over
 Tested with:
 * [Deye SUN-4K-G05](https://www.deyeinverter.com/product/three-phase-string-inverter/sun4-5-6-7-8-10kg03.html) and Logger S/N 23xxxxxxxx
 * [Deye SUN1300G3](https://www.deyeinverter.com/product/microinverter-1/sun13002000g3eu230.html) and Logger S/N 41xxxxxxxx
+* [Deye SUN600G3](https://www.deyeinverter.com/product/microinverter-1/sun600-800-1000g3eu230-single-phase-4-mppt-microinverter-rapid-shutdown.html) and Device S/N 41xxxxxxxx
 
 Supported metrics:
 
 |Metric|Modbus address|MQTT topic suffix|Unit|
 |---|:-:|---|:-:|
 |Production today|0x3c|`day_energy`|kWh|
 |Uptime|0x3e|`uptime`|minutes|
+|Total Production (Active)|0x3F&0x40|`total_energy`|kWh|
+|Daily Production 1|0x41|`dc/pv1/day_energy`|kWh|
+|Daily Production 2|0x42|`dc/pv2/day_energy`|kWh|
+|Daily Production 3|0x43|`dc/pv3/day_energy`|kWh|
+|Daily Production 4|0x44|`dc/pv4/day_energy`|kWh|
+|Total Production 1|0x45&0x46|`dc/pv1/total`|kWh|
+|Total Production 2|0x47&0x48|`dc/pv2/total`|kWh|
```

Comment:
@Antagor Also these two should be renamed to `total_energy`

Btw, why there are only two totals for pv1 and pv2? What about pv3 and pv4?

Reply:
done.

>Btw, why there are only two totals for pv1 and pv2? What about pv3 and pv4?
My documentation don't have it - if you have it in yours we can add it. my micro only have 2.

sensor.py
82        if self.reg_address in registers: 
                 AND    self.reg_address+1 in registers:

Reply:
> My documentation don't have it - if you have it in yours we can add it. my micro only have 2.

Mine does have it. pv3 is at regs 74-75, pv4 is at regs 77-78. These registers are used by 3 phase string inverters and are already mapped. I will address this in a dedicated PR soon.

> sensor.py
82 if self.reg_address in registers:
AND self.reg_address+1 in registers:

This is a very valid point. I've fixed it. Thanks